.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -0,0 +1,35 @@
+---
+name: '🐞 Bug Report'
+about: Create a report to help us improve
+title: ''
+labels: ['type:Bug', 'status:Unconfirmed']
+assignees: ''
+---
+
+<!--
+Found a bug? Please fill out the sections below. 👍
+-->
+
+### Issue Summary
+
+<!--
+A summary of the issue.
+-->
+
+### Steps to Reproduce
+
+1. (for example) Start a new project with `python manage.py runserver`
+2. Edit models.py as follows...
+3. ...
+
+Any other relevant information. For example, why do you consider this a bug and what did you expect to happen instead?
+
+-   I have confirmed that this issue can be reproduced as described on a fresh gymkhana project: (yes / no)
```

Comment:
This should be a checkbox instead of a yes/no answer. Can be put under a category named "Checklist"?